content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -444,6 +444,11 @@ Enables or disables all automatic instrumentation. Setting the environment varia
 **Default**: `false`<br>
 Added in version 1.23.0.
 
+`DD_TRACE_METHODS`
+: List of classes and methods to trace. Accepts a semicolon (`;`) separated list where each entry has the format `TypeName[MethodNames]`, where `MethodNames` is a comma (`,`) separated list of method names. For generic types, replace the angled brackets and the type parameters' names with a backtick (`` ` ``) followed by the number of generic type parameters. For example, `Dictionary<TKey, TValue>` must be written as `` Dictionary`2 ``. For generic methods, you only need to specify the method name. <br>
```

Comment:
I'd say "List of methods to trace" since we only trace methods, not classes. The classes are included here but only to qualify a method (aka the method's full name).

Reply:
In any case, it would be "types and methods," not "classes and methods" ;)

Reply:
Good feedback, added in https://github.com/DataDog/documentation/pull/13522